### DISCUSSION TITLE
Rounded second label

### DISCUSSION THREAD

**Discussion by HalilFocic** [maxGraph/maxGraph#587] [Q&A]
Hello. This example of story shows how to implement more labels for shape:
https://maxgraph.github.io/maxGraph/demo/?path=/story/labels-secondlabel--default

What i need for my project was to initialize this as circle(ellipse). so its rounded. Basically where the yellow box is i need to have the id of cell and i need it to be rounded. I've tried for hours to make it circle but wasn't able to do it. There is probably a catch I am unfamiliar with.

If anyone could help, would appreciate. 🙏

**Comment by tbouffard** [comment]
Hi @HalilFocic did you find a solution for your problem?
AFAIK, the `TextShape` doesn't provide any way to style its background: it's always a rectangle. I suppose the class extension might do the trick, but I don't have time to experiment with that.

**Reply by HalilFocic** [reply]
> Hi @HalilFocic did you find a solution for your problem? AFAIK, the `TextShape` doesn't provide any way to style its background: it's always a rectangle. I suppose the class extension might do the trick, but I don't have time to experiment with that.

I wasn't able to find the solution but I did find something similar in functionality so it was acceptable to client.
